jenkins/chrysalis_atmnbfb.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xle
+
+# boiler: every script must have these three lines
+export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
+export CIME_MACHINE=chrysalis
+source $SCRIPTROOT/util/setup_common.sh
+
+$RUNSCRIPT -j 4 -t e3sm_atm_nbfb -O master --baseline-compare
```

Comment:
Would adding a `--walltime 00:10:00` here be appropriate? These tests are typically fairly short.

Reply:
The CIME wall time manager should adjust the wall times down as these tests run, so it's not necessary.